code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -905,3 +905,18 @@
 		to_chat(M, "<span class='notice'>[tox_message]</span>")
 	. = 1
 	..()
+
+//Apparently we still haven't shaken the use of asbestos into the year 2303
```

Comment:
The current year is actually 2561 unless someone edited something that they shouldn't have.

Reply:
bay lore stuck in my head. also didn't really have any idea when exactly